Sources/App/Migrations.swift

```diff
@@ -29,6 +29,7 @@ class Migrations {
         
         // Sponsors Migrations
         app.migrations.add(SponsorMigrationV1())
+        app.migrations.add(SponsorMigrationV2())
```

Comment:
minor: migrations should be added in the order that they were developed, while I don't think it has any impact here, it's much better practice.

Reply:
Done